QtCollider/widgets/image_painter.h

```diff
@@ -144,7 +144,6 @@ struct ImagePainter
         bool tileVertically = verticalMode == TileVertically;
         bool tileHorizontally = horizontalMode == TileHorizontally;
 
-        qreal x_origin = rect.left();
```

Comment:
that would not be tiling horizontally, that would be drawing the same image repeatedly.